src/main/java/sirius/biz/analytics/reports/LinkCellFormat.java

```diff
@@ -25,14 +25,19 @@ public class LinkCellFormat implements CellFormat {
     protected static final String TYPE = "link";
     protected static final String KEY_URL = "url";
     protected static final String KEY_VALUE = "value";
+    protected static final String KEY_TARGET_BLANK = "targetBlank";
 
     @Override
     public String format(ObjectNode data) {
-        return "<a href=\""
-               + Strings.cleanup(data.path(KEY_URL).asText(), StringCleanup::escapeXml)
-               + "\" classes=\"link\" target=\"_blank\">"
-               + Strings.cleanup(data.path(KEY_VALUE).asText(), StringCleanup::escapeXml)
-               + "</a>";
+        boolean newTab = data.get(KEY_TARGET_BLANK).asBoolean();
+        String html = "<a href=\""
+                      + Strings.cleanup(data.path(KEY_URL).asText(), StringCleanup::escapeXml)
+                      + "\" classes=\"link\" ";
+        if (newTab) {
+            html += "target=\"_blank\" ";
+        }
+        html += ">" + Strings.cleanup(data.path(KEY_VALUE).asText(), StringCleanup::escapeXml) + "</a>";
```

Comment:
**Optional:** This screams string template constant to me 😄 No chance to easily see whats generated here (was hard before the change, too)

Reply:
Please define the meaning of "resolved" in this context @ymo-sci 😇 … IMHO, @sabieber has a valid point here, and at least a comment in this PR's conversation would be nice explaining while it is still left in place.

Reply:
@sbi I agree, but don´t think that it would be much more readable with constants...

Reply:
Did you try locally?

Reply:
...talked past each other - now its much more readable and also link-class works :) @jakobvogel @sabieber 
special thanks @bkositza